crates/ironrdp-server/README.md

```diff
@@ -0,0 +1,3 @@
+# IronRDP Server
+
+Utilities for implementing a RDP server
```

Comment:
nit: other `README`s are not yet very elaborated either, but it would be nice to give a small overview here

Reply:
Agreed, I'll try to add some docs too :)